pom.xml

```diff
@@ -29,7 +29,8 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.3</version>
+    <version>2.12</version>
```

Comment:
Note for other reviewers: from #11 (explained there, and incorporated into this).